.github/workflows/ci-go.yml

```diff
@@ -104,3 +104,44 @@ jobs:
 
       - name: Session tests
         run: make bin/ak test-sessions
+
+  publish_docker_image:
+    needs:
+      - test-sessions
+      - test-runs
+      - test-cli
+      - static-analysis
+      - build-and-test
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    if: github.ref == 'refs/heads/main'
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Setup version info
+        run: |
+          # These are consumed in the Dockerfile.
+          echo "${GITHUB_REF#refs/tags/}" > .version
+          echo "${GITHUB_SHA}" > .commit
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: ${{ secrets.AWS_GITHUB_ROLE }}
+          role-session-name: Github_Action_Release_Autokitteh
+          aws-region: us-east-1
+      - name: Login to Amazon ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v2
+      - name: Build And Push
+        uses: docker/build-push-action@v4
+        with:
+          context: .
+          platforms: linux/amd64
+          tags: ${{ steps.login-ecr.outputs.registry }}/autokitteh:latest
```

Comment:
What's the effect if `latest` here but `stable` in `release.yml`?

Reply:
basically it means every push to main generate a latest version
but once you release, stable tag is updated to the latest release (assuming release is tested)
release also get a tagged version

so push to main generate new ```autokitteh:latest``` image
release tag x.y.z generate ```autokitteh:x.y.z``` and update ```autokitteh:stable``` to match ```autokitteh:x.y.z```

since we can release not the latest commit, latest tag is not updated when we release